org.opent2t.sample.binaryswitch.superpopular/com.contosothings.binaryswitch/js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "opent2t-translator-com-contosothings-binaryswitch",
+  "version": "0.0.2",
+  "description": "ContosoThings Binary Switch",
+  "homepage": "http://opentranslatorstothings.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/opent2t/translators.git"
+  },
+  "main": "thingTranslator.js",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+  "ava": {
+    "files": [
+      "tests/*.js"
+    ]
+  },
+  "dependencies": {
+    "opent2t-translator-com-contosothings-hub": "^0.0.1"
+  },
+  "devDependencies": {
+    "opent2t": "^1.0.4",
```

Comment:
>opent2t [](start = 5, length = 7)

Should be in dependencies now if we want error handling/logging benefits and show case that in samples